experimenter/experimenter/jetstream/models.py

```diff
@@ -201,6 +201,128 @@ def compute_significance(data_point: DataPoint) -> Significance:
 
 
 class ResultsObjectModelBase(BaseModel):
+    def __init__(
+        self,
+        result_metrics: dict[str, set[Statistic]],
+        data: JetstreamData,
+        experiment: NimbusExperiment,
+        window: AnalysisWindow = AnalysisWindow.OVERALL,
+    ):
+        super().__init__()
+
+        assert experiment.reference_branch
+
+        for jetstream_data_point in data:
+            metric = jetstream_data_point.metric
+            statistic = jetstream_data_point.statistic
+
+            if metric in result_metrics and statistic in result_metrics[metric]:
+                comparison_to_branch = jetstream_data_point.comparison_to_branch
+
+                branch_comparison = (
+                    BranchComparison.ABSOLUTE
+                    if jetstream_data_point.comparison is None
+                    else jetstream_data_point.comparison
+                )
+                data_point = DataPoint(
+                    lower=jetstream_data_point.lower,
+                    upper=jetstream_data_point.upper,
+                    point=jetstream_data_point.point,
+                )
+
+                # Need window index for weekly DataPoint objects and for storing
+                # significance for each window. Overall should always be 1 because
+                # there is only ever one overall window.
+                window_index = (
+                    1
+                    if window == AnalysisWindow.OVERALL
+                    else jetstream_data_point.window_index
+                )
+
+                branch = jetstream_data_point.branch
+                branch_obj = getattr(self, branch)
+                branch_obj.is_control = experiment.reference_branch.slug == branch
+
+                group_obj = getattr(
+                    branch_obj.branch_data, METRIC_GROUP.get(metric, Group.OTHER)
+                )
+
+                if metric == Metric.USER_COUNT and statistic == Statistic.PERCENT:
```

Comment:
Not that you need to change it here but in general, if you need to do multiple comparisons like this you can also write it as

```py
(metric, statistic) == (Metric.USER_COUNT, Statistic.PERCENT)
```

and that makes it easier to write for more than 2 comparisons

Reply:
Is that the preferred way to do multiple comparisons in python? It seems less intuitive to me, though not terribly so.

Reply:
Like using tuples/lists for comparisons is very common ie I would write

```py
if x == A or x == B or ....:
```

as 

```py
if x in [A, B, ...]:
```

or the example above of using tuples  to do multiple comparisons, or using sets, so yeah being able to use collection literals in comparisons is very Pythonic.